### DISCUSSION TITLE
[Config Support]: Frigate is working, detecting obejcts (person) but not higligting or creating boxes on live view.

### DISCUSSION THREAD

**Discussion by beze** [blakeblackshear/frigate#14770] [Config Support]
### Describe the problem you are having

Hello Frigate community,

I'm reaching out for help with my current setup involving my HAOS configured to run Frigate. I have a Reolink doorbell camera enabled with RTSP and have set up Go2RTC along with Mosquitto MQTT. My configuration seems correct as events do successfully identify a "person," but I am facing an issue where detected objects are not highlighted or boxed in the live view or recorded movements.

System Information:

Hardware: Beelink EQi12  i3
Camera: Reolink Doorbell WIFI (with RTSP stream enabled)
Software: Frigate (0.14), Home Assistant OS
MQTT Broker: Mosquitto MQTT running successfully
Go2RTC: Configured and working in theory

The camera is streaming correctly via RTSP. ( I have checked on VLC)
Events are correctly identifying a "person."

What Doesn't Work:

There are no bounding boxes or highlights on detected objects in both live view and recorded video.
The detection works, but the visual representation of detected objects is missing.

Another thig is taht if I start go2rtc add on before, the port 8555 gets in use and frigate does not start. 

Troubleshooting Steps Taken:

Verified camera stream using an RTSP player (working correctly).
Checked the Frigate logs for any errors (no relevant errors found).
Restarted Frigate after configuration changes.
Ensured that MQTT is configured correctly and connected.

I would appreciate any guidance or suggestions on how to resolve this issue. Thank you for your support! Sorry for the stupid question, but I am really getting frustrated to not have this working. 

<img width="1203" alt="image" src="https://github.com/user-attachments/assets/4f514c3f-9b25-4fc4-937f-10e5503868fe">

### Version

0.14

### Frigate config file

```yaml
mqtt:
  host: 192.168.5.111
  port: 1883
  user: hamqtt
  password: XX
  topic_prefix: frigate
  client_id: frigate

motion:
  threshold: 30
  lightning_threshold: 0.8
  contour_area: 10

go2rtc:
  streams:
    doorbell: 
      - rtsp://xx:xx@192.168.5.113:554/h264Preview_01_main
      - "ffmpeg:doorbell#audio=aac"

cameras:
  doorbell:
    ffmpeg:
      output_args:
        record: preset-record-generic-audio-copy
      inputs:
        - path: rtsp://127.0.0.1:8554/doorbell
          input_args: preset-rtsp-restream
          roles:
            - detect
            - rtmp  # For streaming with detected objects highlighted
    detect:
      width: 1280
      height: 720
      fps: 30
    live:
      stream_name: doorbell  # Stream name for live view

objects:
  track:
    - person

record:
  enabled: true
  retain: 
    days: 0
    mode: all
  events:
    pre_capture: 5
    post_capture: 5
    objects:
      - person
    retain:
      default: 10
      mode: motion
      objects:
        person: 15

snapshots:
  enabled: True
  bounding_box: True  # Highlight detected objects in snapshots
  retain:
    default: 5
    objects:
      person: 7

version: 0.14
```

### Relevant Frigate log output

```shell
2024-11-04 05:00:07.363764994  [2024-11-04 02:00:07] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2024-11-04 05:00:07.363902232  [2024-11-04 02:00:07] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2024-11-04 05:00:07.364226330  [2024-11-04 02:00:07] ffmpeg.doorbell.detect         ERROR   : [segment @ 0x565162c8b440] Timestamps are unset in a packet for stream 0. This is deprecated and will stop working in the future. Fix your code to set the timestamps properly
2024-11-04 05:00:07.364556898  [2024-11-04 02:00:07] ffmpeg.doorbell.detect         ERROR   : [rtsp @ 0x565162c6e100] RTP: PT=60: bad cseq 032b expected=e769
2024-11-04 05:00:07.364723108  [2024-11-04 02:00:07] ffmpeg.doorbell.detect         ERROR   : [rtsp @ 0x565162c6e100] RTP: PT=61: bad cseq a7be expected=ace8
2024-11-04 05:00:07.364868045  [2024-11-04 02:00:07] ffmpeg.doorbell.detect         ERROR   : [out_1_0 @ 0x5651630b5f80] 100 buffers queued in out_1_0, something may be wrong.
2024-11-04 05:00:07.364985184  [2024-11-04 02:00:07] ffmpeg.doorbell.detect         ERROR   : [out_1_0 @ 0x5651630b5f80] 1000 buffers queued in out_1_0, something may be wrong.
2024-11-04 05:00:07.365105110  [2024-11-04 02:00:07] ffmpeg.doorbell.detect         ERROR   : [out_1_0 @ 0x5651630b5f80] 10000 buffers queued in out_1_0, something may be wrong.
```

### Relevant go2rtc log output

```shell
GO2rtc logs are always like taht, i think I am missing somethingl:

01:38:17.024 INF go2rtc platform=linux/amd64 revision=8cca8de version=1.9.6
01:38:17.024 INF config path=/config/go2rtc.yaml
01:38:17.025 INF [rtsp] listen addr=:8554
01:38:17.025 INF [api] listen addr=:1984
01:38:17.025 INF [webrtc] listen addr=:8555/tcp
```

### Frigate stats

```json
Running
```

### Operating system

HassOS

### Install method

HassOS Addon

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Object Detector

CPU (no coral)

### Screenshots of the Frigate UI's System metrics pages

<img width="1186" alt="image" src="https://github.com/user-attachments/assets/c6c41498-eb99-4bc0-9e44-e09339b06ac3">

**Comment by NickM-27** [comment]
This is working as expected. You need to use the debug live view in settings to see object boxes.

**Reply by beze** [reply]
Hello Nick, thank you so much for your reply!

Could you kindly point me to where I can enable the debug live view? I am not easily finding it. 

Lastly, as shown in the picture I shared, Frigate displays 6 alerts but 0 detections. Is this correct, even though it is "detecting" a person? Thanks!

**Reply by NickM-27** [reply]
> Could you kindly point me to where I can enable the debug live view? I am not easily finding it.

settings cog -> settings -> choose debug in the top tabs

> Lastly, as shown in the picture I shared, Frigate displays 6 alerts but 0 detections. Is this correct, even though it is "detecting" a person? Thanks!

yes, by default people are considered alerts, like it says in the docs.

**Reply by beze** [reply]
Thank you so Much NickM, you made my day! It is really beautfulll to see the model working and doind detection! Congratulations Frigate team!!

Last question: There's some way to enable these bundle boxes on live view? In dashboaard or only on the Debug view?

Cheer and thanks for the incredible job!! Now I will work on the notifications!!

**Reply by NickM-27** [reply]
no, it is only in the debug view

**Comment by beze** [comment]
Thanks for the help!!
